read.go

```diff
@@ -6,15 +6,20 @@ import (
 )
 
 /* Just a helper for reading file with repo URLs. */
-func ReadFileToList(path string) ([]string, error) {
+func ReadFileToList(path string) (lines []string, err error) {
 	file, err := os.Open(path)
 	if err != nil {
 		return nil, err
 	}
-	defer file.Close() // TODO unhandled error
+
+	defer func() {
+		cerr := file.Close()
+		if cerr == nil {
+			err = cerr
+		}
+	}()
```

Comment:
In this code the error is never returned. Looks like a tpyo as your other deferred funcs write this like below: 

```suggestion
	defer func() {
		cerr := file.Close()
		if err == nil {
			err = cerr
		}
	}()
```

Reply:
Ah yes, it is a typo.